templates/launchd.plist

```diff
@@ -73,17 +75,17 @@
     <integer>30</integer>
 
     <key>StandardOutPath</key>
-    <string>/usr/local/var/log/buildkite-agent.log</string>
+    <string>/Users/your-build-user/.buildkite/log/buildkite-agent.log</string>
 
     <key>StandardErrorPath</key>
-    <string>/usr/local/var/log/buildkite-agent.log</string>
+    <string>/Users/your-build-user/.buildkite/log/buildkite-agent.log</string>
 
     <key>EnvironmentVariables</key>
     <dict>
       <key>PATH</key>
       <string>/usr/bin:/bin:/usr/sbin:/sbin:/usr/local/bin</string>
 
-      <key>BUILDKITE_AGENT_TOKEN</key>
```

Comment:
@keithpitt is it meant to match the env var from this [list in the docs](https://buildkite.com/docs/guides/environment-variables)?